http/rights.go

```diff
@@ -9,6 +9,6 @@ import (
 // MakeRightsHandler makes a HTTP handler that returns information about the rights of the user.
 func MakeRightsHandler(authorizationManager security.AuthorizationManager) http.HandlerFunc {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		EncodeReply(r.Context(), w, authorizationManager.GetRightsOfCurrentUser(r.Context()))
+		_ = EncodeReply(r.Context(), w, authorizationManager.GetRightsOfCurrentUser(r.Context()))
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time